js/post-layout.js

```diff
@@ -15,6 +15,11 @@ function replaceInText(element, replacements) {
 var siteUrl = "{{ site.url }}";
 var fontChanged = false;
 var pwd = new PWD();
+
+pwd.on("instanceNew", function(instance){
+  instance.terms[0].write(`$ \r\n`);
```

Comment:
Not necessarily for now (because our guides only have one terminal for now), but I guess we don't want to write a `$` to the first terminal for every instance that gets created?

Reply:
> Not necessarily for now (because our guides only have one terminal for now), but I guess we don't want to write a `$` to the first terminal for every instance that gets created?

Yes. Since this is a temporary fix until we can get the upstream fix merged I think it's good for now. If we're moving forward, we'll probably remove this shortl